app/views/settings/shared/_tabs.html.slim

```diff
@@ -0,0 +1,99 @@
+/# locals: (active_tab:)
+
+ruby:
+  tab_labels = {
+    profile: t("user_accounts.profile"),
+    general: t("company.general"),
+    career_site: t("company.career_site"),
+    email_templates: t("recruitment.email_templates"),
+    sources: t("recruitment.sources"),
+    disqualify_reasons: t("recruitment.disqualify_reasons")
+  }
+
+.col.d-md-none.dropdown.w-100.mobile-nav.border-end.border-bottom
+  .card-body
+    = render ButtonComponent.new( \
+              variant: :cancel,
+              type: :button,
+              flex_content_position: :between,
+              id: "dropdownNavLink",
+              class: "w-100",
+              data: { bs_toggle: :dropdown },
+            ) do |c|
+      - c.with_icon(:chevron_down)
+      span = tab_labels[active_tab]
+    ul.dropdown-menu.w-100 aria-labelledby="dropdownNavLink"
+      h6.dropdown-header
+        = inline_svg_tag("user.svg", { class: "me-2", style: "--tblr-icon-size: 15px" })
```

Comment:
need to discuss icons after answer from Amgalan

Reply:
Fixed in d27d0dd0330ed93f8fe2addc8fef87eeab741e6c